transport/transport.go

```diff
@@ -63,13 +63,11 @@ type recvMsg struct {
 	err error
 }
 
-func (recvMsg) isItem() bool {
-	return true
-}
+func (*recvMsg) isItem() {}
 
 // All items in an out of a recvBuffer should be the same type.
 type item interface {
-	isItem() bool
```

Comment:
I don't understand this comment. This is an unexported interface - what wrapper types are you envisioning?